EventFilter/Phase2TrackerRawToDigi/plugins/Phase2TrackerDumpDigi.cc

```diff
@@ -0,0 +1,209 @@
+#include "DataFormats/Common/interface/Handle.h"
```

Comment:
All classes should have a comment at the top explaining what they are.
Also, doesn't this dump clusters? So why the file name?

Reply:
renamed, comment added